doc/040_backup.rst

```diff
@@ -355,6 +355,7 @@ environment variables. The following list of environment variables:
     RESTIC_REPOSITORY                   Location of repository (replaces -r)
     RESTIC_PASSWORD_FILE                Location of password file (replaces --password-file)
     RESTIC_PASSWORD                     The actual password for the repository
+    RESTIC_PASSWORD_COMMAND		Command printing the password for the repoaitory to stdout
```

Comment:
```suggestion
    RESTIC_PASSWORD_COMMAND		Command printing the password for the repository to stdout
```

Reply:
Beside the small typo, maybe you can also see if you can fix the white space offset and make it align with the other entries?

Reply:
done both